docs/API.md

```diff
@@ -32,7 +32,7 @@ source = <<-EOS
 EOS
 
 reporter = Reek::Report::TextReport.new
-examiner = Reek::Examiner.new(source)
+examiner = Reek::Examiner.new source
```

Comment:
But then we'd should update the rest of our api docs as well. Personally this feels a little like yak-shaving to me i have to say :laughing: 
But if you and @mvz feel that this communicates our intention clearer I'll happily change. 
But right now, I'd rather veto this change ;)